### PR TITLE
Add attribute to ErrorDTO

### LINKED ISSUE #562: AdminPanel HttpUtil EnsureSuccessStatusCode() bug for Schedule
When you put FinishDate less than StartDate.
Exception:  System.NullReferenceException: 'Object reference not set to an instance of an object.'

### PR DESCRIPTION
**Pull request by Tkachelnik**

### CODE REVIEW

**Review by andrii-developer — APPROVED** [review]

**Review by andrii-developer — CHANGES_REQUESTED** [review]
As it was agreed on a meeting. The FluentValidation response model should be converted to our common model.

File: CharlieBackend.Core/DTO/Result/ErrorDto.cs [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ﻿using CharlieBackend.Core.Models.ResultModel;
+using Newtonsoft.Json;
```
Comment (andrii-developer) [inline CharlieBackend.Core/DTO/Result/ErrorDto.cs]: Please remove that using.

File: CharlieBackend.AdminPanel/Middlewares/ExceptionHandleMiddleware.cs [hunk 2]
```diff
@@ -27,8 +28,10 @@ public async Task InvokeAsync(HttpContext context)
             catch (HttpStatusException ex)
             {
                 _logger.LogError($"Something went wrong with API. {ex.Message}");
+                    
+                var encodeMessage = HttpUtility.UrlEncode(ex.Message);
```
Comment (andrii-developer) [inline CharlieBackend.AdminPanel/Middlewares/ExceptionHandleMiddleware.cs]: encodedMessage

File: CharlieBackend.AdminPanel/Controllers/HomeController.cs [hunk 3]
```diff
@@ -14,8 +15,10 @@ public IActionResult Index()
         [Route("Home/ApiError/{statusCode}/{message}")]
         public IActionResult ApiError(uint statusCode, string message)
         {
+            var decodeMessage = HttpUtility.UrlDecode(message);
```
Comment (andrii-developer) [inline CharlieBackend.AdminPanel/Controllers/HomeController.cs]: decodedMessage

**Review by andrii-developer — APPROVED** [review]
Everything is fine. Just a few cosmetic comments. Please address them before the merge.

**Review by Vso1o — APPROVED** [review]

**Review by KSudak — APPROVED** [review]

**Review by Ilhamich — APPROVED** [review]

**Review by DemetriosKoras — APPROVED** [review]

**Review by elizabeth-grant-20 — APPROVED** [review]

**Pull request merged**